Makefile

```diff
@@ -39,13 +38,7 @@ proto:
 		protoc -I $$(dirname $$file) --go_out=plugins=grpc:$$(dirname $$file) $$file; \
 	done
 
-lint: testdeps
-	go get -v github.com/golang/lint/golint
-	for file in $$(git ls-files '*.go' | grep -v '\.pb\.go$$' | grep -v '_string\.go$$'); do \
-		golint $$file; \
-	done
-
-pretest: lint
+pretest:
```

Comment:
On Fri, Jun 12, 2015 at 6:08 PM, David Symonds <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/grpc/grpc-go/pull/225#discussion_r32366036>:
>
> > @@ -39,13 +38,7 @@ proto:
> >  		protoc -I $$(dirname $$file) --go_out=plugins=grpc:$$(dirname $$file) $$file; \
> >  	done
> >
> > -lint: testdeps
> > -	go get -v github.com/golang/lint/golint
> > -	for file in $$(git ls-files '*.go' | grep -v '\.pb\.go$$' | grep -v '_string\.go$$'); do \
> > -		golint $$file; \
> > -	done
> > -
> > -pretest: lint
> > +pretest:
>
> it doesn't seem like you need the pretest target either.
>
It is in a pending commit. I am trying to figure out why travis is broken
on this.

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc-go/pull/225/files#r32366036>.
>